packages/client/tsconfig.json

```diff
@@ -2,6 +2,9 @@
   "extends": "../../tsconfig.json",
   "include": ["src"],
   "compilerOptions": {
-    "types": ["node", "vitest/globals"]
+    "types": ["node", "vitest/globals"],
+    "paths": {
+      "tslib": ["./node_modules/tslib/tslib.d.ts"]
+    }
```

Comment:
why is this needed?

Reply:
I don't remember but some APIs wouldn't work without it. I'll try removing it and see what error pops up to give you an answer.

Reply:
Looks like it's not needed anymore. Must have been for AsyncGenerators / AsyncIterators, which I've removed on the client now.